### ISSUE TITLE
FXMLView#getViewAsync blocking ...

### ISSUE THREAD

**Issue by helmut-at-work** [AdamBien/afterburner.fx#47]
Hi Adam,

playing arround with creating Views I found out that slow initialize methods (Thread.sleep(xxx)) in the view classes block the Platform Thread.
I assume that the  getViewAsync(Consumer<Parent> consumer) methode should build the node in background and only add the Parent in the Platform Thread. Therfore I changed the methode to:

```java
public void getViewAsync(Consumer<Parent> consumer) {
		PARENT_CREATION_POOL.execute(() -> {
			Parent v = getView();
			Platform.runLater(() -> consumer.accept(v));
		});
	}
```
Right?

**Comment by RAnders00** [comment]
:+1:

**Comment by AdamBien** [comment]
What about:

```java
    public void getViewAsync(Consumer<Parent> consumer) {
        Supplier<Parent> supplier = this::getView;
        CompletableFuture.supplyAsync(supplier).thenAcceptAsync(consumer, FX_PLATFORM_EXECUTOR);
    }
```
Is available with: 1.6.3-SNAPSHOT

**Comment by AdamBien** [comment]
Works well, tested with followme.fx. Slightly changed to: 

    public void getViewAsync(Consumer<Parent> consumer) {
        Supplier<Parent> supplier = this::getView;
        CompletableFuture.supplyAsync(supplier, PARENT_CREATION_POOL).
                thenAcceptAsync(consumer, FX_PLATFORM_EXECUTOR);
    }
Improvement is available in:

```java

    public void getViewAsync(Consumer<Parent> consumer) {
        Supplier<Parent> supplier = this::getView;
        CompletableFuture.supplyAsync(supplier, PARENT_CREATION_POOL).
                thenAcceptAsync(consumer, FX_PLATFORM_EXECUTOR);
    }
```
